img/client-smile.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="xMidYMid" width="80" height="49" viewBox="0 0 80 49">
+  <image xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAAxCAMAAAB6Z95gAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAABMlBMVEUAAAD///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////8AAAAtaWdIAAAAZHRSTlMAEJ+/rzDvQGB/zyDfcAEEIUIZQZAKpaEsg5GABsmNTf14whbAluI0A6L5dQKnviUkm/Z7eZp8/ttZTMxRI83VZwwF+vKZPi/H5JThxY8rBwhv4/zdqIh3XcothnO1J1WC1+ru4fibwwAAAAFiS0dEAIgFHUgAAAAJcEhZcwAACxIAAAsSAdLdfvwAAAHgSURBVEjH7ZdZW4JAFIbRFLTFrFzKsLSscG13Ky1LbU/TMrXUsvP/f0OOUAGK4jAX1eN39TFzeDmcMw8zUNQg6fQTXRmMUo8tAwiidRKPLfgWI/Fj4H8C0iIITQJoEkFMJICU0dzRJA8Re21iRBDmVwJR9ab6eFwZO/2d7uPxNGPWo/VikntcMfxCMci9RqBeJ/da6kcbmF4/1t+ShTRu1kqSZ52lqLl5crz5BZSkzU6KZ7d1C2h0EKqjxbnImyUXGaJrWTCse4UEb8XNftkpWnuOFhst+pKvehwsPqv7lg7PqvjaC2vrWnjrPvBKRzZgcwuft7UJG7Ihzg+BYAgPFwoGwMnJR7kwQGQbh7cTAQhzvePcLsDe/si9YQ/2AA65vnPRGEA8MdICsiTiALGo0nQSbbtHx6pLGTo+Qn8FSeWIVBrtlCenGTW4zOkJik6nBgWdmc9RUDaXH5JmKJ/Loshz89mQ515c8geEq+ukYp6Z5PUVH3V5oabON8JJNXB7l7gvSGcL94m724AQcKOyg8WH0s8Jvfzoe3JVnjuquJ58j+WfmdJDURUOia3WYIhq1RGXbP0lrkyLv9RHown9eW1ke2HZxquKTiiJa7be/JH39ofH89F+j/jfWk1u8B2fd1LIz6HHr60AAAAASUVORK5CYII=" width="80" height="49"/>
+</svg>
```

Comment:
Looks like you're encoding a PNG in an SVG? Why? Just use the PNG.

Also, doesn't look like this file is being used; delete it.